MTPot.py

```diff
@@ -170,8 +253,23 @@ def main():
     except MissingConfigField:
         honey_logger.info("Syslog reporting disabled, to enable it add its configuration to the configuration file")
     COMMANDS = config.commands
-    server = gevent.server.StreamServer((config.ip, config.port), MyTelnetHandler.streamserver_handle)
-    honey_logger.info("Listening on %d...", config.port)
+
+    try:
+        the_timeout = config.timeout
+    except MissingConfigField:
+        the_timeout = default_timeout
+
+    try:
+        OVERWRITE_COMMANDS = config.overwrite_commands
```

Comment:
The problem is that the object _MyTelnetHandler_  and particularly the `@command(OVERWRITE_COMMANDS_LIST)...` is instanced during the class declaration. So, these commands must be declared before (see line 19).
Lines 211 and 214 are then useless.